Source/WebCore/platform/graphics/win/FontCustomPlatformDataWin.cpp

```diff
@@ -107,4 +108,10 @@ bool FontCustomPlatformData::supportsFormat(const String& format)
         || equalLettersIgnoringASCIICase(format, "woff"_s);
 }
 
+bool FontCustomPlatformData::supportsTech(const FontTech& tech)
+{
+    // FIXME: define supported techonlogies for this platform (webkit.org/b/256310).
```

Comment:
(typo)

I think we should populate this fully before landing the patch. The maintainers of the other ports probably can just tell you what this function should say. `return true` is harmful, as it will cause unrecognized fonts to be marked as recognized on those ports.